revpi-hat-PR100328R03.json

```diff
@@ -9,11 +9,265 @@
     "dtstr": "revpi-flat-s-2022",
     "gpiobanks": [
         {
-            "drive": "default",
+            "drive": "8mA",
             "slew": "default",
             "hysteresis": "default",
             "gpios": [
-
+                {
+                    "gpio": 2,
+                    "fsel": "alt0",
+                    "pull": "none",
+                    "comment": [ "SDA1 (I2C1.SDA)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 3,
+                    "fsel": "alt0",
+                    "pull": "none",
+                    "comment": [ "SCL1 (I2C1.SCL)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 4,
+                    "fsel": "alt4",
+                    "pull": "down",
+                    "comment": [ "TXD3 (UART3.TX)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 5,
+                    "fsel": "alt4",
+                    "pull": "none",
+                    "comment": [ "RXD3 (UART3.RX)" ]
+                },
+                {
+                    "gpio": 6,
+                    "fsel": "input",
+                    "pull": "none",
+                    "comment": [ "GPIO (SPI6.nCS0)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 7,
+                    "fsel": "input",
+                    "pull": "none",
+                    "comment": [ "GPIO (UART3.RTS)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 8,
+                    "fsel": "alt4",
+                    "pull": "none",
+                    "comment": [ "TXD4 (UART4.TX)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 9,
+                    "fsel": "alt4",
+                    "pull": "none",
+                    "comment": [ "RXD4 (UART4.RX)" ]
+                },
+                {
+                    "gpio": 10,
+                    "fsel": "input",
+                    "pull": "none",
+                    "comment": [ "GPIO (SPI6.nCS1)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 11,
+                    "fsel": "input",
+                    "pull": "none",
+                    "comment": [ "GPIO (UART4.RTS)",
+                                 "external pull-up" ]
+                },
+                {
+                    "gpio": 13,
+                    "fsel": "alt3",
+                    "pull": "none",
+                    "comment": [ "SPI5_MISO (SPI5.MISO)",
+                                 "external pull-down" ]
```

Comment:
What about without the comma, so "external pull-down to configure the KSZ8795CLXIC to SPI client mode". This is a better scheme in case that multiple clients use pull downs/ups.

Reply:
This is okay for me as well